tests/lib/rules/no-implicit-globals.js

```diff
@@ -412,7 +412,92 @@ ruleTester.run("no-implicit-globals", rule, {
         // This rule doesn't disallow assignments to properties of readonly globals
         "Array.from = 1;",
         "Object['assign'] = 1;",
-        "/*global foo:readonly*/ foo.bar = 1;"
+        "/*global foo:readonly*/ foo.bar = 1;",
+
+
+        //------------------------------------------------------------------------------
+        // exported
+        //------------------------------------------------------------------------------
+
+        // `var` and functions
+        "/* exported foo */ var foo = 'foo';",
+        "/* exported foo */ function foo() {}",
+        {
+            code: "/* exported foo */ function *foo() {}",
+            parserOptions: { ecmaVersion: 2015 }
+        },
+        {
+            code: "/* exported foo */ async function foo() {}",
+            parserOptions: { ecmaVersion: 2017 }
+        },
+        {
+            code: "/* exported foo */ async function *foo() {}",
+            parserOptions: { ecmaVersion: 2018 }
+        },
+        "/* exported foo */ var foo = function() {};",
+        "/* exported foo */ var foo = function foo() {};",
+        {
+            code: "/* exported foo */ var foo = function*() {};",
+            parserOptions: { ecmaVersion: 2015 }
+        },
+        {
+            code: "/* exported foo */ var foo = function *foo() {};",
+            parserOptions: { ecmaVersion: 2015 }
+        },
+        "/* exported foo \n* exported bar */ var foo = 1, bar = 2;",
```

Comment:
```suggestion
        "/* exported foo, bar */ var foo = 1, bar = 2;",
```

This format is not supported. This directive would be [parsed](https://github.com/eslint/eslint/blob/e940be7a83d0caea15b64c1e1c2785a6540e2641/lib/linter/config-comment-parser.js#L39) without errors, but the variables it declares as exported actually are: `["foo", "*", "exported", "bar"]`.

(the same for other tests with multiple exported variables)